src/cryptoadvance/specter/static/styles.css

```diff
@@ -908,38 +909,46 @@ a:focus + .tool-tip .tool-tip__info {
 @media (max-width: 1100px){
   #side-content{
     position: absolute;
+    display: flex;
     height: 100%;
-    left: -235px;
+    left: -250px;
+    width: 270px;
     transition: left 0.3s;
     box-shadow: 3px 0px 10px rgba(0,0,0,0.3);
   }
-  #side-expand{
+  #side-content:after{
     display: flex;
+    content: "▶";
     align-items: center;
     justify-content: center;
-    font-size: 14px;
+    font-size: 12px;
     cursor: pointer;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    width: 20px;
+    background: var(--cmap-bg-ligher);
   }
   td.xpub{
     max-width: 230px;
   }
   @media (hover: hover){
     main{
-      padding-left: 15px;
+      padding-left: 20px;
     }
     #side-content:hover{
-      position: absolute;
-      left: 0px;
+      left: 0;
     }
-    #side-content:hover > #side-expand{
-      transform: scaleX(-1);
+    #side-content:hover:after{
+      right: -20px;
```

Comment:
```suggestion
      position: absolute;
      right: -20px;
```

Reply:
It already is positioned absolutely via [line 926](https://github.com/cryptoadvance/specter-desktop/pull/394/files/ae2fc0ddd2007f952f5292ad61847a8c4bf451fd#diff-51788d3f42cf0703ce17eb762b3b4f1bR926).

Reply:
Yes but now that it's sticky the arrow remains visible even on hover (unless that's intended?)

Reply:
You're right, thanks again.